src/components/calcite-value-list/calcite-value-list.e2e.ts

```diff
@@ -43,4 +43,54 @@ describe("calcite-value-list", () => {
   describe("disabled states", () => {
     disabledStates("value");
   });
+
+  describe.only("drag and drop", () => {
+    let page: E2EPage;
+    beforeEach(async () => {
+      page = await newE2EPage();
+      await page.setContent(`<calcite-value-list multiple drag-enabled>
+        <calcite-value-list-item value="one" text-label="One"></calcite-value-list-item>
+        <calcite-value-list-item value="two" text-label="Two"></calcite-value-list-item>
+        <calcite-value-list-item value="three" text-label="Three"></calcite-value-list-item>
+      </calcite-value-list>`);
+    });
+    it.skip("works using a mouse", async () => {
+      // TODO: remove skip once https://github.com/GoogleChrome/puppeteer/issues/1376 addressed
```

Comment:
See link for why this test won't work - there's a bug. Leaving it in but skipping.